src/org/zaproxy/zap/extension/pscan/PassiveScanParam.java

```diff
@@ -155,4 +165,30 @@ public void setScanOnlyInScope(boolean scanOnlyInScope) {
     public boolean isScanOnlyInScope() {
         return scanOnlyInScope;
     }
+    
+    /**
+     * Sets whether or not the passive scan should be performed on traffic
+     * from the fuzzer.
+     *
+     * @param scanFuzzerMessages
+     *            {@code true} if the scan should be performed on traffic
+     *            generated by the fuzzer, {@code false} otherwise.
+     * @since TODO Add version
+     * @see #isScanFuzzerMessages()
+     */
+    public void setScanFuzzerMessages(boolean scanFuzzerMessages) {
+        this.scanFuzzerMessages = scanFuzzerMessages;
+        getConfig().setProperty(SCAN_FUZZER_MESSAGES_KEY, Boolean.valueOf(scanFuzzerMessages));
```

Comment:
The (explicit) boxing should be removed (per new guidelines).

Reply:
:+1: